courses-app/src/components/CreateCourse/CreateCourse.jsx

```diff
@@ -1,14 +1,141 @@
 import "./CreateCourses.css";
 import Button from "../../common/Button/Button";
-const CreateCourse = () => {
-  const Duration = "00:00";
+import { mockedAuthorsList } from "../../constants";
+import { v4 as uuidv4 } from "uuid";
+import { useState } from "react";
+import { courses } from "../../constants";
+const CreateCourse = ({
+  setIsCreateOpened,
+  isCreateOpened,
+  setCoursesList,
+  coursesList,
+}) => {
+  const { v4: uuidv4 } = require("uuid");
+  const [minutes, setMinutes] = useState();
+  const [authors, setAuthors] = useState(mockedAuthorsList);
+  const [author, setAuthor] = useState();
+  const [authorList, setAuthorList] = useState([]);
+  const [course, setCourse] = useState({
+    name: "",
+    text: "",
+    authors: [],
+    time: 0,
+    date: 0,
+    id: "",
+  });
+
+  const enterTitle = (value) => {
+    setCourse({ ...course, name: value });
+  };
+  const enterDescription = (value) => {
+    setCourse({ ...course, text: value });
+  };
+  const Duration = ({ minutes }) => {
+    if (!minutes) {
+      return (
+        <h1>
+          Duration: <span>{"00:00"}</span> hours
+        </h1>
+      );
+    }
+    const num = Number.parseInt(minutes);
+    const hours = num / 60;
+    let rhours = Math.floor(hours);
+    const minut = (hours - rhours) * 60;
+    let rminut = Math.floor(minut);
+    rhours = rhours < 9 ? "0" + rhours : rhours;
+    rminut = rminut < 9 ? "0" + rminut : rminut;
+    const time = rhours + ":" + rminut;
```

Comment:
Move this part of the code outside the CreateCourse component. For example, create the new file with the name "helper.js" and store the function there